spec/ndt7-protocol.md

```diff
@@ -62,12 +62,18 @@ NOT be smaller than 1 << 17 bytes.
 
 Both textual and binary WebSocket messages are allowed. Textual WebSocket
 messages will contain serialised JSON structures containing measurements
-results (see below). When downloading, the server is expected to send
-measurement to the client, and when uploading, conversely, the client is
-expected to send measurements to the server. Both parties SHOULD NOT
-send measurement results when they're not expected to. Measurements SHOULD
-NOT be sent more frequently than every 250 ms, to avoid generating too
-much unnecessary processing load on the receiver.
+results (see below). Both the client and the server MAY send measurements
+messages during any subtest. Upon receiving an unexpected measurement
+message, a party SHOULD discard such message and SHOULD continue
```

Comment:
How is `unexpected measurement` defined here? Both client and server may send measurements at any time, and the case "messages are sent too quickly" is detailed separately below. Maybe it should be `malformed message`?

Reply:
Right, this is ambiguous. Let me address it.

Reply:
I've pushed better text that should explain exactly what would happen. Could you read the whole diff again and let me know if there is additional ambiguity? (When we concur that the text is okay, then a bigger job is probably to have a conversation on whether we want the WebSocket to be full or half duplex during a ndt7 experiment.)